src/main/java/me/spartacus04/jext/integrations/IntegrationsManager.kt

```diff
@@ -19,9 +20,29 @@ class IntegrationsManager {
         integrations.forEach { registerIntegration(it) }
     }
 
-    fun hasJukeboxAccess(player: Player, block: Block): Boolean = integrations.all { it.hasJukeboxAccess(player, block) }
+    fun hasJukeboxAccess(player: Player, block: Block): Boolean? {
+        for(integration in integrations){
+            val state = integration.hasJukeboxAccess(player, block)
+            return when(state){
+                true->true
+                false->continue
+                null->null
+            }
+        }
+        return false
+    }
 
-    fun hasJukeboxGuiAccess(player: Player, block: Block): Boolean = integrations.all { it.hasJukeboxGuiAccess(player, block) }
+    fun hasJukeboxGuiAccess(player: Player, block: Block): Boolean? {
+        for(integration in integrations){
+            val state = integration.hasJukeboxGuiAccess(player, block)
+            return when(state){
+                true->true
+                false->continue
+                null->null
+            }
+        }
+        return false
+    }
```

Comment:
Since the methods are not nullable you can do the old `integrations.all` function